<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="login_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="edt_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="pwd_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="pwd_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="acc_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="acc_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>账号</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="login_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_9">
      <widget class="QPushButton" name="sig_button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reg_button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_10">
      <widget class="QFrame" name="chk_frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="name_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>姓名</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="ret_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chk_button">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>我是老师</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>我是学生</string>
     </property>
    </widget>
    <zorder>stackedWidget</zorder>
    <zorder>pwd_frame</zorder>
    <zorder>acc_frame</zorder>
    <zorder>login_label</zorder>
    <zorder>checkBox</zorder>
    <zorder>checkBox_2</zorder>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>xx:xx:xx</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="deco_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>跑得快不一定赢，
不跌跟头才是成功 
         --8848钛金手机</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="hello_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>晚上好呀！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
